modules/social_features/social_activity/config/install/views.view.activity_stream_notifications.yml

```diff
@@ -360,17 +360,15 @@ display:
       display_extenders: {  }
       path: notifications
       pager:
-        type: full
+        type: mini
         options:
-          items_per_page: 10
+          items_per_page: 30
```

Comment:
Changed this to 30 as well. I think that makes more sense considering you already see 10 items in the notification bar.

Reply:
Nice addition!